src/Calculator/Calculator.vcxproj.filters

```diff
@@ -328,6 +328,9 @@
       <Filter>Common</Filter>
     </ClCompile>
     <ClCompile Include="Common\ViewState.cpp" />
+    <ClCompile Include="ApplicationViewModeTrigger.cpp">
+      <Filter>Views\StateTriggers</Filter>
+    </ClCompile>
```

Comment:
We should probably move ApplicationViewModeTrigger.cpp to the views folder and have the file structure line up with the project structure.